internal/services/tem/domain.go

```diff
@@ -226,7 +226,7 @@ func ResourceDomainRead(ctx context.Context, d *schema.ResourceData, m interface
 	_ = d.Set("next_check_at", types.FlattenTime(domain.NextCheckAt))
 	_ = d.Set("last_valid_at", types.FlattenTime(domain.LastValidAt))
 	_ = d.Set("revoked_at", types.FlattenTime(domain.RevokedAt))
-	_ = d.Set("last_error", domain.LastError)
+	_ = d.Set("last_error", domain.LastError) //nolint:staticcheck
```

Comment:
We need to add test and documentation to ensure that we detect when this field is removed from API responses and that the user are warned that the field is deprecated.